alpaca/broker/client.py

```diff
@@ -601,3 +634,251 @@ def download_trade_document_for_account_by_id(
+            bank_id (Union[UUID, str]): The ID of the Bank to delete.
+        """
+        account_id = validate_uuid_id_param(account_id)
+        bank_id = validate_uuid_id_param(bank_id, "bank_id")
+        self.delete(f"/accounts/{account_id}/recipient_banks/{bank_id}")
+
+    def create_transfer_for_account(
+        self,
+        account_id: Union[UUID, str],
+        transfer_data: Union[CreateACHTransferRequest, CreateBankTransferRequest],
+    ) -> Transfer:
+        """
+        Creates a single Transfer for the given account.
+
+        Args:
+            account_id (Union[UUID, str]): The ID of the Account to create the bank connection for.
+            transfer_data (Union[CreateACHTransferRequest, CreateBankTransferRequest]): The request data used to
+              create the bank connection.
+
+        Returns:
+            Transfer: The Transfer that was created.
+        """
+        account_id = validate_uuid_id_param(account_id)
+        response = self.post(
+            f"/accounts/{account_id}/transfers", transfer_data.to_request_fields()
+        )
+        return Transfer(**response)
+
+    def get_transfers_for_account(
+        self,
+        account_id: Union[UUID, str],
+        transfers_filter: Optional[GetTransfersRequest] = None,
+        max_items_limit: Optional[int] = None,
+        handle_pagination: Optional[PaginationType] = None,
+    ) -> Union[List[Transfer], Iterator[List[Transfer]]]:
+        """
+        Gets the transfers for an account.
+
+        Args:
+            account_id (Union[UUID, str]): The ID of the Account to create the bank connection for.
+            transfers_filter (Optional[GetTransferRequest]): The various filtering parameters to apply to the request.
+            max_items_limit (Optional[int]): A maximum number of items to return over all for when handle_pagination is
+              of type `PaginationType.FULL`. Ignored otherwise.
+            handle_pagination (Optional[PaginationType]): What kind of pagination you want. If None then defaults to
+              `PaginationType.FULL`.
+
+        Returns:
+            Union[List[Transfer], Iterator[List[Transfer]]]: Either a list or an Iterator of lists of Transfer child
+              classes.
+        """
+        account_id = validate_uuid_id_param(account_id)
+        handle_pagination = BrokerClient._validate_pagination(
+            max_items_limit, handle_pagination
+        )
+
+        iterator = self._get_transfers_iterator(
+            account_id=account_id,
+            transfers_filter=transfers_filter
+            if transfers_filter is not None
+            else GetTransfersRequest(),
+            max_items_limit=max_items_limit,
+        )
+
+        return BrokerClient._return_paginated_result(iterator, handle_pagination)
+
+    def _get_transfers_iterator(
+        self,
+        account_id: UUID,
+        transfers_filter: Optional[GetTransfersRequest],
```

Comment:
Since we ensure to always pass at least an empty one in, and we never check for `None` we should probably just make this not optional

Reply:
Fixed.